lib/nolibc/errno.c

```diff
@@ -32,9 +32,15 @@
  */
 
 #include <errno.h>
+#include <stddef.h>
 
 #if CONFIG_HAVE_SCHED
 __thread int errno;
 #else
 int errno;
 #endif /* !CONFIG_HAVE_SCHED */
+
+int *__errno_location(void)
+{
+	return NULL;
```

Comment:
Shouldn't this point to errno?
```suggestion
	return &errno;
```

Reply:
I think so, too. In such a case, we have an actual implementation and not a stub anymore 😄